more/free-programming-cheatsheets.md

```diff
@@ -289,7 +291,9 @@
 
 ### Machine Learning
 
+* [Cheat Sheet of Machine Learning and Python (and Math) Cheat Sheets](https://medium.com/machine-learning-in-practice/cheat-sheet-of-machine-learning-and-python-and-math-cheat-sheets-a4afe4e791b6) - Medium (HTML)
```

Comment:
We can also credit the author here, Robbie Allen.

Reply:
done